src/pages/Groups/Group4/styles/Screen2.styled.ts

```diff
@@ -0,0 +1,42 @@
+import styled from 'styled-components'
+
+export const ContainerDiv = styled.div`
+  background-color: #ffffff;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`
+
+export const ForgetPasswordContainer = styled.div`
+  height: 100vh;
+  width: 100vw;
+  margin: 0px 23px;
+  padding-top: 50px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`
+
+export const Logo = styled.img`
+  width: 70%;
+  max-width: 255px;
+  max-height: 255px;
+  margin-top: 20vh;
+`
+
+export const Link = styled.a`
+  width: 370px;
+  height: 19px;
+  left: 22px;
+  top: 467px;
```

Comment:
unnecessary since the position is not absolute or fixed

'left', 'top', 'right' and 'bottom' are only needed if you are fixing the position of the element. if not its not useful